### ISSUE TITLE
Upgrade to latest Gradle wrapper

### ISSUE THREAD

**Issue by yairm210** [yairm210/Unciv#4945]
tl;dr:

- [x] Upgrade Packr logic to work with latest Packr version (4.0.0) - [Success!](https://github.com/yairm210/Unciv/runs/3394537484) - so we can
- [x] Remove dependency on JAVA_HOME in Desktop Gradle, so we can
- [x] Upgrade machine to Java 11 - [Success!](https://github.com/yairm210/Unciv/runs/3394582270) - so we can
- [x] Upgrade Gradle - [Success!](https://github.com/yairm210/Unciv/actions/runs/1156353957)

Currently I'm at the point where I can't use the new Packr classes from within Gradle the way I did with the old ones, since there's a usage of Apache commons CompressorStreamFactory.detect(), which Gradle *refuses* to acknowledge exists.

The current plan is, as Anuken/Mindustry does, to run packr **from the command line**. Works locally on my Ubuntu.
This means that we can migrate gradually, and have multiple versions, unlike if we use the classes.
Won't work for Windoge 32 so we'll have to decide between remaining with the current config for that (if it works on with Java 11, which it should) or giving up on Windows 32 entirely. If need bhe we'll go the second route, but as long as the cost isn't high, I'm planning on sticking with the first.
